lib/rules/border-zero.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
             var node = item.content[0];
             if (node.type === 'number' || node.type === 'ident') {
               if (node.content === '0' || node.content === 'none') {
-                if (parser.options.convention !== node.content) {
```

Comment:
Hey @richarddewit - Thanks for the PR! Just one thing - while the solution is valid, could we be a bit more descriptive and use `.toString()` instead?